README.md

```diff
@@ -29,19 +29,15 @@
 
 These steps are to be completed by the author of the blog post.
 
-1. Create [an issue](https://github.com/OpenLiberty/blogs/issues/new) for the blog post. This is to help the editors track the progress of the post.
+1. Create an issue for the blog post. This is to help the editors track the progress of the post.
 
    * **All blog posts except release blog posts**
 
-     Create a normal issue. In the description, give a simple outline of the purpose of the blog post. If there is a specific date by which the post must be available, mention that in the description too.
+     Create [an issue](https://github.com/OpenLiberty/blogs/issues/new). In the description, give a simple outline of the purpose of the blog post. If there is a specific date by which the post must be available, mention that in the description too.
 
-   * **GA release blog posts** (Open Liberty GA release announcements only)
+   * **GA and beta release blog posts** (Open Liberty GA and beta release announcements only)
 
-     Create an issue using the `Open Liberty release notes` issue template. Make sure to select each task in the issue as you complete it to show progress.
-
-   * **Beta release blog posts** (Open Liberty beta release announcements only)
-
-     Create a normal issue. In the title, make clear that it's for the beta release and which release version.
+     Create a new issue using the `Open Liberty release notes` issue template. Make sure to select each task in the issue as you complete it to show progress.
```

Comment:
is this still correct now that the GA and beta release posts are at least partially automated through GitHub actions? I'm not sure of the extent of that automation of if it makes this step obsolete

Reply:
Hmm, I don't know what the process is now. @mbroz2 should probably comment...

Reply:
haha, I just had a conversation with Reece earlier this week that I need to update documentation for the release/beta blog posts.  It's on my to-do, but I've also been waiting for the dust to settle with regards to GHA so I don't need to continuously update the documentation.

Reply:
in the meantime, don't worry about this section as I've been doing all the recent release posts.